apps/web/src/services/asset-inventory/collector/shared/collector-forms/collector-form-store.ts

```diff
@@ -21,6 +21,7 @@ export const useCollectorFormStore = defineStore('collector-form', {
         scheduleHours: [] as number[],
         schedulePower: false,
         attachedServiceAccount: null as AttachedServiceAccount|null,
+        pluginMetadata: {} as Record<string, any>,
```

Comment:
nitpick: `any` << Is this temp type?

Reply:
suggestion: Please change this type to JsonSchema from @spaceone/design-system.